style.css

```diff
@@ -463,38 +553,13 @@
   }
 }
 
-/* Desctop */
-
-@media only screen and (min-width: 768px) {
-  .nav-container {
-    background-color: #3a3c39;
-    display: flex;
-    justify-content: space-between;
-  }
+/* Desktop */
 
-  .burger {
+@media only screen and (min-width: 776px) {
+  hr {
     display: none;
   }
 
-  .name {
-    margin-left: 9.625rem;
-  }
-
-  .shortcut-contener {
-    margin-right: 12.875rem;
-    align-self: center;
-  }
-
-  .shortcut-mail {
-    margin-bottom: -0.5625rem;
-  }
-
-  .shortcut {
-    margin-right: 2rem;
-    color: white;
-    text-decoration: none;
-  }
-
   .home-container {
     background-color: #262626;
```

Comment:
- [ ] [optional] I would suggest removing the empty white margin from your webpage's top and left side, It will make your project looks more professional.

### see the image for reference 

![image](https://user-images.githubusercontent.com/83803414/167854667-43e27706-efa1-4328-a031-4a62894782c5.png)